src/components/assets-dialog.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button } from "@/components/ui/button";
+import { Dialog, DialogClose, DialogContent, DialogDescription, DialogFooter, DialogHeader, DialogTitle } from "@/components/ui/dialog";
+import { ArrowUpRightIcon, BrickWallIcon, CoinsIcon, DiamondIcon, FlagIcon, HeartPulseIcon, PersonStandingIcon, SkullIcon, SwordIcon } from "lucide-react";
+
+interface CellGroup {
+  label: string
+  items: CellItem[]
+}
+
+interface CellItem {
+  label: string
+  icon: any,
```

Comment:
Can be typed as `LucideIcon` to be more specific (needs the import).

```suggestion
  icon: LucideIcon,
```